train_hierarchialXC.py

```diff
@@ -42,15 +77,50 @@
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Dataloader initialization ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Optimizer?

Reply:
Should I create a complete `my_neural_XC` for the parameters as in [this](https://github.com/vishwakftw/extreme-classification/blob/master/train_neuralxc.py#L116)?

Reply:
apologizes for all the childish questions. :)

Reply:
No no. This would how the optimizer would look like:
```
optimizer = getattr(torch.optim, opt_options['name'])(ae.parameters(),
**opt_options['args'])
```